featurex/converters/api.py

```diff
@@ -29,6 +30,7 @@ def __init__(self, api_key=None):
         self.recognizer = sr.Recognizer()
         self.api_key = api_key
 
+    @abstractmethod
```

Comment:
This is to help https://github.com/tyarkoni/featureX/pull/73/files#diff-5bd92ddf7c5112f06b5c4d480ed14af3R93. Get converter will/should only give back non-abstract converters. SpeechRecognitionConverter shouldn't be initialized as a concrete class anyways, since it'd break without a valid `environ_key` or `recognize_method`.